pkg/config/apm.go

```diff
@@ -58,6 +58,7 @@ func setupAPM(config Config) {
 	config.BindEnvAndSetDefault("apm_config.windows_pipe_security_descriptor", "D:AI(A;;GA;;;WD)", "DD_APM_WINDOWS_PIPE_SECURITY_DESCRIPTOR") //nolint:errcheck
 	config.BindEnvAndSetDefault("apm_config.remote_tagger", false, "DD_APM_REMOTE_TAGGER")                                                    //nolint:errcheck
 
+	config.BindEnvAndSetDefault("debugging.enable_remote_configuration", false) //nolint:errcheck
```

Comment:
I think this goes with the other group above. Also, this seems like it's in the wrong place in this file. Lastly, isn't the main key `live_debugging`? Is it `debugging`?